docs/configure-artifact-repository.md

```diff
@@ -25,12 +25,15 @@ The actual repository used by a workflow is chosen by the following rules:
 
 ## Configuring MinIO
 
+You can install MinIO into your cluster via Helm.
+
+First, [install `helm`](https://helm.sh/docs/intro/install/). Then, install MinIO with the below commands:
+
 ```bash
-brew install helm # mac, helm 3.x
 helm repo add minio https://helm.min.io/ # official minio Helm charts
 helm repo update
 helm install argo-artifacts minio/minio --set service.type=LoadBalancer --set fullnameOverride=argo-artifacts
-```
+\```
```

Comment:
GitHub can't render code blocks in code blocks, so the `\` escape ends up visible. Could you manually remove the `\`?

Reply:
Sorry.. I should have reviewed your proposal, but I couldn't. I'll fix it right away

Reply:
The result is as follows
<img width="710" alt="image" src="https://github.com/argoproj/argo-workflows/assets/39519869/7187eadc-a7a5-402d-b26b-91ca983e9260">

Reply:
> Sorry.. I should have reviewed your proposal, but I couldn't. I'll fix it right away

no worries, it's a GitHub issue unfortunately that pops up often with docs code blocks

> The result is as follows

LGTM!